<launch>
    <node name="usb_cam_right_wrist" pkg="usb_cam" type="usb_cam_node" output="screen" >
      <param name="video_device" value="/dev/CAM_RIGHT_WRIST" />
      <param name="framerate" value="60" />
      <param name="image_width" value="640" />
      <param name="image_height" value="480" />
      <param name="pixel_format" value="yuyv" />
      <param name="camera_frame_id" value="usb_cam" />
      <param name="io_method" value="mmap"/>
      <param name="autofocus" value="false"/>
      <param name="focus" value="40"/>
      <param name="autoexposure" value="true"/>
  </node> 

  <arg name="serial_no" default="313522071698" />
  <arg name="topic_name" default="usb_cam_high/image_raw" />
  
  <group ns="usb_">
    <include file="$(find realsense2_camera)/launch/rs_camera.launch">
      <!--<arg name="serial_no" value="$(arg serial_no)" />-->
      <arg name="align_depth" value="false" />
      <arg name="enable_pointcloud" value="false" />
      <arg name="color_width" value='640'/>
      <arg name="color_height" value='480'/>
      <arg name="color_fps" value='30'/>


    </include>
  </group>
  <!--
  <node name="image_processor" pkg="easy" type="image.py" output="screen"/>
  <node name="realsense_node" pkg="easy" type="realsense_node.py" output="screen"/>
  -->

</launch>